DOCS/interface-changes.rst

```diff
@@ -27,6 +27,7 @@ Interface changes
       ("") uses the value of --hwdec.
     - drop deprecated --ad/--vd features
     - drop deprecated --sub-codepage syntax
+    - add --ontop-level option for macOS
```

Comment:
i would also rather keep it since i changed the default behaviour. anyway i fixed the merge conflict for now.